packages/multi-provider/tests/index.test.ts

```diff
@@ -25,78 +22,82 @@ describe('multi-provider', async () => {
   const testSigner = testProvider.getSigner();
 
   // register A and B domains
-  before('registers domains', () => {
+  beforeAll(() => {
     mp = new MultiProvider<Domain>();
     mp.registerDomain(chainADomain);
     mp.registerDomain(chainBDomain);
   });
 
   it('returns an array of domains', () => {
     const values = mp.registeredDomains;
-    expect(values).to.include(chainADomain);
-    expect(values).to.include(chainBDomain);
+    expect(values).toContainEqual(chainADomain);
+    expect(values).toContainEqual(chainBDomain);
```

Comment:
What's the difference between toContain and toContainEqual?

Reply:
`toContain` is used for checking if a value in the array is strictly equal to the passed in value. This breaks down for objects depending on how the test is set up where you might not want strict equality (`===`). For this specific test, we really want to test the structure of the object, which we can use `toContainEqual` for.